hivemq-edge/src/frontend/src/locales/en/translation.json

```
@@ -552,8 +552,8 @@
       "formats": {
         "label_CONFIGURATION": "Full configuration",
         "description_CONFIGURATION": "This will export the whole configuration of the adapter, as a standalone JSON file.",
-        "label_SUBSCRIPTIONS": "Subscriptions",
-        "description_SUBSCRIPTIONS": "This will only export the subscriptions currently defined in the adapter, in a format suitable for batch processing.",
+        "label_MAPPINGS": "Mappings",
+        "description_MAPPINGS": "This will only export the mappings currently defined in the adapter, in a format suitable for batch processing.",
```

suggestion(non-blocking): A bit hard to get my head around it. WDYT about the following?

```suggestion
        "description_MAPPINGS": "Export all defined adapter mappings in a batch processing format. (CSV)?",
```